tests/test_init.py

```diff
@@ -366,6 +364,54 @@ async def test_entry_rename_object_count_sensor(hass: HomeAssistant) -> None:
         )
 
 
+async def test_entry_cleanup_old_camera_snapshot(hass: HomeAssistant) -> None:
+    """Test cleanup of old camera snapshot."""
+    entity_registry = er.async_get(hass)
+
+    config_entry: MockConfigEntry = MockConfigEntry(
+        entry_id=TEST_CONFIG_ENTRY_ID,
+        domain=DOMAIN,
+        data={CONF_HOST: "http://host:456"},
+        title="Frigate",
+        version=2,
+    )
+
+    config_entry.add_to_hass(hass)
+
+    old_unique_ids = {
+        ("camera", f"{TEST_CONFIG_ENTRY_ID}:camera:front_door_person"),
+        ("sensor", f"{TEST_CONFIG_ENTRY_ID}:sensor_fps:front_door_camera"),
+        ("sensor", f"{TEST_CONFIG_ENTRY_ID}:sensor_object_count:front_door_person"),
+        ("sensor", f"{TEST_CONFIG_ENTRY_ID}:sensor_fps:detection"),
+        ("sensor", f"{TEST_CONFIG_ENTRY_ID}:sensor_fps:front_door_process"),
+        ("sensor", f"{TEST_CONFIG_ENTRY_ID}:sensor_fps:front_door_skipped"),
+        ("switch", f"{TEST_CONFIG_ENTRY_ID}:switch:front_door_recordings"),
+    }
+
+    # Create fake entries with the old unique_ids.
+    for platform, unique_id in old_unique_ids:
+        assert entity_registry.async_get_or_create(
+            platform, DOMAIN, unique_id, config_entry=config_entry
+        )
+
+    # Setup the integration.
+    config_entry = await setup_mock_frigate_config_entry(
+        hass, config_entry=config_entry
+    )
+
+    removed_unique_ids = {
+        ("camera", f"{TEST_CONFIG_ENTRY_ID}:camera:front_door_person"),
```

Comment:
Why doesn't this contain a `camera_snapshots` unique id? (I thought this was the whole point of this testing method, to make sure the old snapshot cameras were removed)?

Reply:
... and perhaps check the new image entity has been added?

Reply:
yeah I think that was a mistake on my part